stories/Atom/Global/Icons/icon.stories.mdx

```diff
@@ -0,0 +1,241 @@
+import { IconsTimesClose } from './icon_Times_close'
+import { IconsSearch} from './search'
+import { IconsLanguage} from './Language'
+import { IconsGlobe} from './globe'
+import { IconsCalendar} from './calender'
+import { Iconscircleplay} from './circle_play'
+import { Iconscircleup} from './circle_up'
+import { Iconscircledown} from './circle_down'
+import { Iconscircleleft} from './circle_left'
+import { Iconscircleright} from './circle_right'
+import { Iconschevrondown} from './chevron_down'
+import { Iconschevronup} from './chevron_up'
+import { Iconschevronleft} from './chevron_left'
+import { Iconschevronright} from './chevron_right'
+import { Iconschevronsmleft } from './chevron_small_left'
+import { Iconschevronsmright} from './chevron_small_right'
+import { Iconschevronsmup } from './chevron_small_up'
+import { Iconschevronsmdown} from './chevron_small_down'
+import { Iconarrowright} from './arrow_right'
+import { Iconarrowleft} from './arrow_left'
+import { Iconarrowleftexpand} from './arrow_left_expand'
+import { Iconexternallink} from './external_link'
+import { Icondownload} from './download'
+
+<Meta title="Atoms/Icons"  />
+
+<style>{`
+  .subheading {
+    font-weight: 700;
+    font-size: 18px;
+    color: #666;
+    letter-spacing: 1.5px;
+    line-height: 24px;
+    margin-bottom: 12px;
+    margin-top: 40px;
+  }
+  div[id^="story--atoms-icons"] {
+    display: inline-flex;
+    width: 140px;
+    height: 50px;
+    align-items: center;
+    margin: 10px;
+    border: 1px solid #000;
+    vertical-align: top;
+    justify-content: center;
+  }
+  .subheading2 {
+    font-weight: 700;
+    font-size: 14px;
+    color: #666;
+    letter-spacing: 1.5px;
+    line-height: 24px;
+    margin-bottom: 12px;
+  } 
+`}</style>
+
+# Icons Component
+Icons can grab attention and draw users in to a piece of content. They can help users quickly understand the context in which they are placed. 
+
+<div className="subheading2">When to use</div>
+
+Use icons where the content is complex and dense to help users comprehend the available actions.
+Icons can be used in place of long words or phrases, saving space.
+Use Icons that are recognisable and accompany corresponding text labels where possible to make sure the users understand the context.
+
+
+<ul>
+<li>Icons are used in all over the section component.</li>
+<li>Icons animation done by the css</li>
+<li>icons are saved in <code>assets/images/Icons</code> folder in SVG format.</li>
+</ul>
+
+
+<div className="subheading">How to Use</div>
+<ul>
+<li>We can use the <code>icon</code> in the any component by import the related<code>Icon atom</code> in the component code.</li>
+<li>We can also use through the SCSS by <code>background tag </code> property.</li>
+</ul>
+
+
+<Story name="IconsBar">
+  {(args, { globals: { locale } }) => {
+    return <IconsBar></IconsBar> ;
+  }}
+</Story>
+
+<Story name="IconsTimes">
+  {(args, { globals: { locale } }) => {
```

Comment:
@harishkumar-srijan Please remove args and globals as they are not required and same as in all the icons stories

Reply:
Doen